google/cloud/bigquery/job/query.py

```diff
@@ -1430,40 +1430,6 @@ def _reload_query_results(
             timeout=transport_timeout,
         )
 
-    def _done_or_raise(self, retry=DEFAULT_RETRY, timeout=None):
```

Comment:
This was overridden because we wanted `result()` from the superclass to call `jobs.getQueryResults`, not just `jobs.get` (i.e. `job.reload()` in Python). Now that we aren't using the superclass for `result()`, this method is no longer necessary.